odin/api/v1/core/views.py

```
@@ -80,6 +87,17 @@ def get(self, request: Request, *args: list, **kwargs: dict) -> HttpResponse:
         return HttpResponse(contents, content_type="image/svg+xml")
 
 
+class DashboardView(APIView):
+    authentication_classes = []
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)